setup.examples/postfix/mailwatch-postfix.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Configuration script for mailwatch with postfix
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
```

Comment:
mailwatch-postfix.sh
Only this to do in main.cf to have Postfix running "out of box"?